data/projects/l/lifinance.yaml

```diff
@@ -16,70 +17,267 @@ blockchain:
+    tags:
+      - safe
+  - address: "0x8BCC385948C73736423D38cc567cFEdE0F1826A3"
+    networks:
+      - matic
+    tags:
+      - safe
+  - address: "0xdf61270fDC1A892874Fd3C0143A0A4CBA74F4EF1"
+    networks:
+      - mode
+    tags:
+      - safe
+  - address: "0xa8892eA3fdDeF2aa8AfB1E3643a3284f978A5114"
+    networks:
       - optimism
     tags:
-      - contract
-      - factory
-    name: unverified
-  - address: "0x28c6039ef5634a9e1f6a45014ee9c5b5c64d580f"
+      - safe
+  - address: "0xdf61270fDC1A892874Fd3C0143A0A4CBA74F4EF1"
+    networks:
+      - linea
+    tags:
+      - safe
+  - address: "0x02f1272aEaCaf7BD8b30278bc2AA381Cc623A744"
+    networks:
+      - zksync_era
+    tags:
+      - safe
+  - address: "0x9575B9fC42dec56D3772B3df5DA047a1f1D55582"
+    networks:
+      - polygon_zkevm
+    tags:
+      - safe
+  - address: "0xdf61270fDC1A892874Fd3C0143A0A4CBA74F4EF1"
+    networks:
+      - scroll
+    tags:
+      - safe
+  - address: "0xa89a87986e8ee1Ac8fDaCc5Ac91627010Ec9f772"
+    networks:
+      - mantle
+    tags:
+      - safe
+
+  # Create Factories
+  - address: "0x93FEC2C00BfE902F733B57c5a6CeeD7CD1384AE1"
     networks:
       - arbitrum_one
+      - base
+      - mainnet
+      - matic
+      - mode
       - optimism
+      - polygon_zkevm
+      - scroll
+      - mantle
     tags:
       - contract
       - factory
-    name: unverified
-  - address: "0x2f16c2c6467377920c125c05b7adbc5d74ae5fa1"
+    name: CREATE3Factory
+  - address: "0x8437A5fE47A4Df14700c96DF1870824e72FA8499"
     networks:
-      - arbitrum_one
-      - optimism
+      - linea
     tags:
       - contract
       - factory
-    name: unverified
+    name: CREATE3Factory
+
+  # Diamonds
   - address: "0x1231deb6f5749ef6ce6943a275a1d3e7486f4eae"
     networks:
       - arbitrum_one
+      - base
+      - mainnet
+      - matic
+      - mode
+      - optimism
+      - polygon_zkevm
+      - scroll
+      - mantle
+    tags:
+      - contract
+    name: LiFiDiamond
+  - address: "0x341e94069f53234fe6dabef707ad424830525715"
+    networks:
+      - zksync_era
+    tags:
+      - contract
+    name: LiFiDiamond
+  - address: "0xde1e598b81620773454588b85d6b5d4eec32573e"
+    networks:
+      - linea
     tags:
       - contract
     name: LiFiDiamond
   - address: "0x9b11bc9fac17c058cab6286b0c785be6a65492ef"
     networks:
       - arbitrum_one
+      - mainnet
+      - matic
+      - optimism
+      - polygon_zkevm
     tags:
       - contract
     name: LiFiDiamondImmutable
   - address: "0x362fa9d0bca5d19f743db50738345ce2b40ec99f"
     networks:
       - arbitrum_one
+      - mainnet
+      - matic
+      - optimism
     tags:
       - contract
     name: LiFiDiamond
+
+  # FeeCollectors
   - address: "0xb0210de78e28e2633ca200609d9f528c13c26cd9"
     networks:
       - arbitrum_one
+      - base
+      - mainnet
+      - matic
+      - mode
+      - optimism
+      - polygon_zkevm
+      - scroll
+      - mantle
     tags:
       - contract
```

Comment:
This FeeCollector only appears as a contract on arbitrum_one. Please remove the other network tags (unless I am misunderstanding how this contract works cross chain).

Reply:
Updated.